README.md

```diff
@@ -9,12 +9,16 @@ Read the [overview](https://kueue.sigs.k8s.io/docs/overview/) to learn more.
 
 ## Production Readiness status
 
-- ✔️ Coverage with unit and integration tests: ~72%
+- ✔️ API version: v1beta1, respecting [Kubernetes Deprecation Policy](https://kubernetes.io/docs/reference/using-api/deprecation-policy/)
 - ✔️ Up-to-date [documentation](https://kueue.sigs.k8s.io/docs).
-- ✔️ Monitoring via [metrics](https://kueue.sigs.k8s.io/docs/reference/metrics)
-- ✔️ API validation and defaulting.
-- ⏳ Scalability tests in progress.
-- ✔️ API version: v1beta1, with backwards-compatibility guarantees.
+- ✔️ Good Test Coverage:
+  - ✔️ Unit Test: ~77.2% (controllers are excluded for they're covered by integration tests)
```

Comment:
Actually, it could be useful to link to testgrids:
- https://testgrid.k8s.io/sig-scheduling#periodic-kueue-test-unit-main
- https://testgrid.k8s.io/sig-scheduling#periodic-kueue-test-integration-main
- https://testgrid.k8s.io/sig-scheduling#periodic-kueue-test-e2e-main-1-26

Reply:
The testgrid is mostly valuable for developers, I don't think normal visitors will show interest with this.

Reply:
If they see a fully green testgrid, it could be useful :)

But I'm fine either way